src/main/java/marquez/db/JobRunDao.java

```diff
@@ -68,4 +69,15 @@ default void updateState(UUID jobRunID, Integer state) {
           + " ON (jr.guid = :guid AND jr.job_run_args_hex_digest = jra.hex_digest) "
           + "WHERE jr.guid = :guid")
   JobRun findJobRunById(UUID guid);
+
+  @SqlQuery(
+      "SELECT jr.*, jra.args_json "
+          + "FROM job_runs jr "
+          + "INNER JOIN job_versions jv "
+          + " ON (jr.job_version_guid = jv.guid AND jv.job_guid = :jobUuid)"
+          + "LEFT JOIN job_run_args jra "
+          + " ON (jr.job_run_args_hex_digest = jra.hex_digest) "
```

Comment:
_Minor_: Just realized we need to update the table schemas for `job_runs` and `job_runs_args` (which drops `job_run_args_hex_digest`, etc)

- Update job_run_args schema #541
- Update job_runs schema #542

Reply:
good reminder, thanks